packages/mds-agency/types.ts

```diff
@@ -11,8 +11,7 @@ export type AgencyApiRequest<P extends Params = ParamsDictionary> = ApiRequest<P
 
 export type AgencyApiAccessTokenScopes = 'admin:all' | 'vehicles:read'
 
-export type AgencyApiResponse<TBody = any> = ApiVersionedResponse<
-  AGENCY_API_SUPPORTED_VERSION,
+export type AgencyApiResponse<TBody = any> = ApiResponse<
```

Comment:
Also, remove the `| {}` that was added to the end of the declaration of `ApiReponse`